src/mesh/NodeDB.cpp

```diff
@@ -195,6 +195,13 @@ void NodeDB::installDefaultConfig()
     config.position.position_broadcast_smart_enabled = true;
     config.position.broadcast_smart_minimum_distance = 100;
     config.position.broadcast_smart_minimum_interval_secs = 30;
+
+    config.position.channel_precision_count = 8;
+    config.position.channel_precision[0] = 32;
```

Comment:
Technically, we shipped the protobufs themselves on the last release. Should we create a migration of sorts to init this for the first time?

Reply:
Ironically we've come to a bit different conclusion on how this should work. Will be dropping some of this work in favor of that approach.